### PR TITLE
Adding ResourceQuota and SchedulerPreemption test as they passed for …

### PR DESCRIPTION
**Pull request by immuzz**
…Windows QoS

### CODE REVIEW

**Comment by immuzz** [comment]
@PatrickLang Did a PR for https://github.com/kubernetes/kubernetes/issues/73418. Please review

**Comment by immuzz** [comment]
I have signed the CLA with Linux Foundation (using Microsoft account) as a Microsoft employe and attached my Github account. Let me know if you need anything else

**Comment by neolit123** [comment]
@immuzz you need to sign the CLA if not already.

/check-cla

**Comment by adelina-t** [comment]
/hold

@immuzz Please also increase the job timeout with the average time it takes for these jobs to run. We are already at the 7 hour ( !!! ) mark on average for the job.

File: config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml [hunk 1]
```diff
@@ -69,6 +69,6 @@ periodics:
       - "--acsengine-win-binaries"
       - "--acsengine-agentpoolcount=3"
       - "--test_args=--node-os-distro=windows --ginkgo.focus=\\[Conformance\\]|\\[NodeConformance\\]|\\[sig-windows\\]|\\[sig-apps\\].CronJob|\\[sig-api-machinery\\].ResourceQuota|\\[sig-scheduling\\].SchedulerPreemption --ginkgo.skip=\\[LinuxOnly\\]|\\[k8s.io\\].Pods.*should.cap.back-off.at.MaxContainerBackOff.\\[Slow\\]\\[NodeConformance\\]|\\[k8s.io\\].Pods.*should.have.their.auto-restart.back-off.timer.reset.on.image.update.\\[Slow\\]\\[NodeConformance\\]"
-      - "--timeout=450m"
+      - "--timeout=460m"
```
Comment (adelina-t) [inline config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml]: I would add 20 to account for flakes.

Comment (adelina-t) [reply config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml]: You should also add 20 mins to line 51.

L51 is the timeout for the job, L72 is the timeout for kubetest. The job timeout should be 20 mins greater than the one for kubetest.

**Comment by immuzz** [comment]
@adelina-t @PatrickLang Added 10 minutes of timeout and committed again. ResourceQuota tests take about 4 minutes to pass and SchedulerPreempter about 3 minutes to pass.

File: config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml (hunk 1 above)
Comment (immuzz) [reply config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml]: I changed both. Thanks @adelina-t . Its ready for review!

**Comment by immuzz** [comment]
@adelina-t @PatrickLang Added 20 minutes of timeout to the test and the job. Ready for review

**Comment by PatrickLang** [comment]
I think you forgot to save, or missed one of the merge conflicts. Can you rebase off master, then squash the commits and fix the merge when you do that?

**Comment by immuzz** [comment]
@PatrickLang Rebased and squashed the commits. I merged it again with right version. Please review now.

**Comment by jsturtevant** [comment]
@adelina-t @PatrickLang could you give the /lgtm label?  I tried but do not have the permissions

**Pull request merged**